config/config.js

```diff
@@ -0,0 +1,37 @@
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const development = {
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DEVELOP_DB_NAME,
+    host: "127.0.0.1",
```

Comment:
I repeat again why passing static string value for the host, you should use process.env here.

Reply:
I thought it was only for the database only.

But I just corrected the host value and for the dialect as well, if that's okay